.github/workflows/pre-submit.e2e.generic.workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: pre-submit e2e generic workflow
```

Comment:
Note: the last part of the filename is typically for config option

Reply:
Do you think I should leave it off? Or maybe give it another name? I guess I just put something there to have the same number of file name parts.

Reply:
If you think it should be changed I'll send a follow up PR.